utils/mstpctl-utils-functions.sh

```diff
@@ -6,6 +6,7 @@ mstpctl_parse_ports()
     # For compatibility: the 'all' option.
     case $1 in
       all)
+	# shellcheck disable=SC2121
 	shift &&
 	set regex eth.\* em.\* 'p[0-9].*' noregex "$@"
```

Comment:
`set` is being used here to replace the positional parameters (to replace $@ with some new values).  Usually there should be two dashes after `set` when using it to replace the positional parameters: `set -- regex ...`
I haven't tested, but it looks like shellcheck will not warn about this if there are two dashes after set, so maybe try adding those?